### DISCUSSION TITLE
Spacy format change from iob/conll/conllu

### DISCUSSION THREAD

**Discussion by algo-sroy** [explosion/spaCy#8377] [Help: Coding & Implementations] [answered]
I am trying to convert the dataset to .Spacy format for v3.0.
The command I am using is from the reference : 
python -m spacy convert ./datasets/sample.txt ./datasets -c conllu -s

Can someone tell me what is the purpose of -n 10? It says you may want to group sentences into documents with  '-n 10' .

The error that I am getting is :
ValueError: not enough values to unpack (expected 10, got 2)
Any help is appreciated!

**Comment by adrianeboyd** [comment, marked as answer]
iob/conll/conllu are three different formats. For NER-only data, use `-c ner` to have it try to autodetect (probably `conll` is the right one). You can see examples of all the support NER formats here: https://github.com/explosion/spaCy/tree/master/extra/example_data/ner_example_data

Some of the data formats mark document and/or sentence boundaries and some don't. If your format does not have document boundaries, then it's useful to use `-n 10` or similar to group sentences into paragraph-sized docs for training, since training only on single sentences can lead to degraded performance on longer texts. This is mainly important for the parser, but it can also make a difference for NER if you expect longer texts in real use but your training data would otherwise be converted into single-sentence docs.

**Reply by algo-sroy** [reply]
Thanks for the help!
